third_party/freedom-typings/freedom-common.d.ts

```diff
@@ -102,7 +113,8 @@ declare module freedom {
     // We use this specification so that you can reference freedom sub-modules by
     // an array-lookup of it's name. One day, maybe we'll have a nicer way to do
     // this.
-    [moduleName:string] : Function;
+    // TODO: explore how to use FreedomModuleFactoryManager.
+    [moduleName:string] : any;
```

Comment:
OK, github was shwoing confusing diff and @soycode and I misread stuff. I just discussed and we concluded that this line should not be here. Being removed. :)